package.json

```diff
@@ -31,6 +31,7 @@
     "watch": "node tasks/watch.js",
     "lint": "eslint --version && eslint .",
     "lint-fix": "eslint . --fix || true",
+    "no-new-func": "eslint --debug --no-ignore --no-eslintrc --rule '{no-new-func: error}' $(ls dist/plotly*.js | grep -v plotly-locale* | grep -v plotly-gl* | grep -v plotly-with-meta.* | grep -v plotly.js | grep -v plotly.min.js)",
```

Comment:
Nice! Probably don't need debug long-term, right? And I feel like we might as well include the locales in this test, for completeness... we know they're safe but wouldn't hurt to prove it.